operations/business-operations/customer-support.md

```diff
@@ -2,27 +2,91 @@ Customer Support is a core function at Mattermost with a goal to ensure the succ
 
-If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can [click here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
+## Existing Users of Mattermost Support ##
+
+If you’ve worked with the Mattermost Support Team before and have existing or previous tickets with Support then you already have an account with the email address that you would have used.  To (re)set your password, you can click [here](https://mattermost.zendesk.com/auth/v2/login/password_reset?auth_origin=327479%2Ctrue%2Ctrue&brand_id=327479&return_to=https%3A%2F%2Fsupport.mattermost.com%2Fhc%2Fen-us&theme=hc).
+
+## Ticket Management ##
+
+## Ticket Creation ##
+
+When submitting a Support Ticket to the Mattermost Support Team we recommend that you use the ‘Submit a Request’ option from with the Support Portal.  This will ensure that we get the required information from you.
+
+## Support Ticket Request Form ##
+
+To ensure that we can assist our customers most efficiently it’s imperative that as much information is provided when submitting the ticket.  For details of what details should be provided you should review our Troubleshooting Guide which can be found [here](https://docs.mattermost.com/guides/administrator.html#troubleshooting).
+
+|Field | Description |
+|------|-------------|
+| **CC**  |Add additional email addresses of those who you would like to receive notifications of this issue.|
+| **Subject** |A brief description of the issue.            |
+| **Priority**     |Urgent, High, Normal, or Low, descriptions of these priority ratings can be found here.              |
+| **Category**     |Select a category that best fits the product area affected by the issue raised.            |
+| **Description**     |Provide as much detail as possible to enable the Support Team to recreate the issue.  Please refer to the troubleshooting guide for what’s required.            |
+| **Environment**     |Select the environment where you are experiencing the issue.            |
+| **Mattermost Version**     |Enter the number of the Mattermost server version you are currently running.            |
+| **Mattermost Mobile version**     |If the issue you are reporting is on a mobile device, enter the version here.            |
+| **Mobile device**     |Please select which type of device you are experiencing the issue on.            |
+| **Attachments**     |Add screenshots, logs, etc to the ticket here.            |
+
+ ## Ticket Workflow ##
+
+While working with our customers the Support Team will us the following ticket statuses to manage the ticket, New, Open, Pending, Solved, and Closed.  The workflow will be as follows:
```

Comment:
```suggestion
The Support Team use the following ticket statuses to manage customer tickets: New, Open, Pending, Solved, and Closed. The workflow will be as follows:
```
- Is the workflow going to be added or are we referring to the steps/content that follows this sentence? 
- Are the ticket statuses listed referring to UI elements in Zendesk (asking with regard to formatting the words if needed)?

Reply:
1. Yes, I'm working with Jason to upload the images now.
2. Yes